app/client/sagas/creditTransferSagas.ts

```diff
@@ -49,7 +46,10 @@ function* updateStateFromCreditTransfer(result: CreditLoadApiResult) {
 
   const transfer_accounts = normalizedData.entities.transfer_accounts;
   if (transfer_accounts) {
-    yield put({ type: DEEP_UPDATE_TRANSFER_ACCOUNTS, transfer_accounts });
+    yield put({
+      type: TransferAccountActionTypes.DEEP_UPDATE_TRANSFER_ACCOUNTS,
+      transfer_accounts
+    });
```

Comment:
I think this should be an action... `TransferAccountAction.deepUpdateTransferAccounts()`